modules/setting/ui.go

```diff
@@ -34,6 +34,7 @@ var UI = struct {
 	SearchRepoDescription bool
 	OnlyShowRelevantRepos bool
 	ExploreDefaultSort    string `ini:"EXPLORE_PAGING_DEFAULT_SORT"`
+	EditorEol             string `ini:"EDITOR_EOL"`
```

Comment:
I guess this `ini` tag here is unecssary, but I do like this expliciteness so that the name stays greppable.

Reply:
I think the correct Go name would even be `EditorEOL`